test/integration/fixtures/retries/nested.js

```diff
@@ -0,0 +1,9 @@
+describe('retries', function() {
+  this.retries(3);
+  describe('nested', function () {
+    it('should retry on test 3', function(){
```

Comment:
The name of this test is a little ambiguous. This test will have only a single retry, and will fail on both attempts